Unix/etc/omiserver.conf

```diff
@@ -72,3 +72,6 @@
 ## if USER is specified within <> as the service account, 
 ##      it will be replaced with $USER when copied to output*
 service=<USER>
+
+## Enabling this will cause each provider to run under its own agent
+#individualAgent=false
```

Comment:
This isn't a good name. This is more a debugging feature to help debug provider crashes in omiagent. How about something like `agentDebugging`?

I'd also like to see a much better description for this. Perhaps something like:

```
# Enabling this will cause each provider to run under it's own omiagent
# process. This will take considerably more memory, but is useful for
# diagnosing omiagent cores due to a provider fault. Setting of `true`
# means that each provider runs under it's own omiagent process.
#agentDebugging=false

Reply:
I wasn't crazy about my choice of name anyway, so agentDebugging it is!

Reply:
Adding Paul to reviewer list.